index.bs

```diff
@@ -696,10 +696,16 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     1. If <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> [=list/is not empty=], execute a 
         platform-specific procedure to determine which, if any, credentials in
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> are present on this |authenticator| by matching with 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/id}}</code> and 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/type}}</code>, and set |credentialList| to
-        this filtered list.
+        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> are present on this |authenticator| by
```

Comment:
D'oh: In the "is empty" case, the "also have a type within scopedOptions.allowList." bit shouldn't be there, since there are no types in an empty allowList.